packages/client/src/runtime/utils/datmodelBuilder.ts

```diff
@@ -0,0 +1,55 @@
+import { DMMF } from '@prisma/generator-helper'
+
+import { BaseDMMFHelper } from '../dmmf'
+
+export function field(kind: DMMF.FieldKind, name: string, type: string, extra?: Partial<DMMF.Field>): DMMF.Field {
```

Comment:
Is this a runtime util or a test util?

Reply:
Test util. I was not sure where to put it. Same `runtime/utils` folders contains `getTestClient` and friends as well, but we can create a new `testUtils` folder somewhere and move them there.